packages/abnormal_security/data_stream/vendor_case/manifest.yml

```
@@ -0,0 +1,79 @@
+title: Vendor Case Logs
+type: logs
+streams:
+  - input: cel
+    title: Vendor Case Logs
+    description: Collecting Vendor Case logs via API.
+    template_path: cel.yml.hbs
+    vars:
+      - name: initial_interval
+        type: text
+        title: Initial Interval
+        multi: false
+        required: true
+        show_user: true
+        default: 24h
+        description: How far back to pull the Vendor Case logs from Abnormal Security API. Supported units for this parameter are h/m/s.
+      - name: interval
+        type: text
+        title: Interval
+        description: Duration between requests to the Abnormal Security API. Supported units for this parameter are h/m/s.
+        default: 5m
+        multi: false
+        required: true
+        show_user: true
+      - name: page_size
+        type: text
+        title: Page Size
+        multi: false
+        required: true
+        show_user: false
+        description: Page size for the response of the Abnormal Security API.
+        default: 100
+      - name: http_client_timeout
+        type: text
+        title: HTTP Client Timeout
+        description: Duration before declaring that the HTTP client connection has timed out. Supported time units are ns, us, ms, s, m, h.
+        multi: false
+        required: true
+        show_user: false
+        default: 30s
+      - name: enable_request_tracer
+        type: bool
+        title: Enable request tracing
+        multi: false
+        required: false
+        show_user: false
+        description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging. See [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-cel.html#_resource_tracer_filename) for details.
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: false
+        default:
+          - forwarded
+          - abnormal_security-vendor-cases
```

I think the standard would be `package_name-data_stream`.
```suggestion
          - abnormal_security-vendor_cases
```